overlord/hookstate/hookmgr.go

```diff
@@ -143,13 +153,38 @@ func (m *HookManager) doRunHook(task *state.Task, tomb *tomb.Tomb) error {
 		return err
 	}
 
-	// TODO: Actually dispatch the hook.
+	_, err = RunHook(setup.Snap, setup.Revision, setup.Hook)
+	if err != nil {
```

Comment:
Yeah agreed on the error handling. I'll investigate that method as well, thank you!